app/views/statuses/_status.html.haml

```diff
@@ -17,7 +17,7 @@
 - if status.reply? && include_threads
   - if @next_ancestor
     .entry{ class: entry_classes }
-      = link_to_more ActivityPub::TagManager.instance.url_for(@next_ancestor)
+      = link_to_older ActivityPub::TagManager.instance.url_for(@next_ancestor)
```

Comment:
Hm, in case of threads, I am not sure if older/newer makes more sense than just "more". It's true that descendants are "newer" and ancestors are "older", but it may still be confusing. I'm not sure. Would love to hear what others think.

Reply:
I think it's good either way, though showing just “more” would require yet another translatable string.